packages/playground/windows/playground/MainPage.h

```diff
@@ -17,11 +17,13 @@ struct MainPage : MainPageT<MainPage> {
   Windows::Foundation::Collections::IVector<Microsoft::ReactNative::IReactPackageProvider> m_packageProviders;
 
   bool m_useWebDebugger{false};
+  winrt::hstring debugHost;
```

Comment:
looks like the other member of this file is using m_ for the members. -- You should at least be consistent with the rest of the members.

Reply:
My bad, I also went ahead and removed the the other variable since it wasn't used.